pydatastructs/miscellaneous_data_structures/tests/test_stack.py

```diff
@@ -1,10 +1,18 @@
 from pydatastructs.miscellaneous_data_structures import Stack
 from pydatastructs.linear_data_structures import OneDimensionalArray
 from pydatastructs.utils.raises_util import raises
+from pydatastructs.utils.misc_util import _check_type
 
 def test_Stack():
+    s = Stack(implementation='array')
+    s1 = Stack()
+    assert type(s) is type(s1)
+    s2 = Stack(implementation='linked_list')
+    assert _check_type(s2,Stack) is True
```

Comment:
```suggestion
    assert _check_type(s2, Stack) is True
```

Reply:
How will we check if that `s2` is `LinkedListStack`?
Probably something like,
`_check_type(s2, Stack) and _check_type(s2, LinkedListStack)` should work well.

Reply:
I have made some changes please review it again.